diff --git a/es/select/src/Select.js b/es/select/src/Select.js
index bbb31fb43f353cc625d417ec83effe5b00bea2da..ae526c20150d0470cd7f05aba15425a4ba6a6966 100644
--- a/es/select/src/Select.js
+++ b/es/select/src/Select.js
@@ -616,6 +616,10 @@ export default defineComponent({
             blur: () => {
                 var _a;
                 (_a = triggerRef.value) === null || _a === void 0 ? void 0 : _a.blur();
+            },
+            focusInput: () => {
+                var _a;
+                (_a = triggerRef.value) === null || _a === void 0 ? void 0 : _a.focusInput();
             }
         };
         const cssVarsRef = computed(() => {
diff --git a/es/select/src/interface.d.ts b/es/select/src/interface.d.ts
index 00dd9ae918e7b5ebd4713f36ce446f241b92e615..98985a1526ec47515bdf399547cb4bd41a2c66e2 100644
--- a/es/select/src/interface.d.ts
+++ b/es/select/src/interface.d.ts
@@ -45,6 +45,7 @@ export type SelectGroupOption = (SelectGroupOptionBase & {
 export interface SelectInst {
     focus: () => void;
     blur: () => void;
+    focusInput:() => void
 }
 export type SelectFallbackOption = (value: string & number) => SelectOption;
 export type SelectFallbackOptionImpl = (value: string | number) => SelectOption;